ydb/Cargo.toml

```diff
@@ -16,6 +16,7 @@ force-exhaustive-all=[] # The feature disable all non_exhaustive attributes in y
 async-trait="0.1"
 derivative="2"
 derive_builder="0.12.0"
+futures = "0.3"
```

Comment:
The sdk has dependency from tokio already. Use second async runtime may cause additional hard debug problems.

It is a good approach as first step - for implement "work concept".
Second step - remove the additional dependency and use already present dependency - tokio.